src/main/java/org/example/springbootintro/dto/book/BookSearchParametersDto.java

```diff
@@ -0,0 +1,4 @@
+package org.example.springbootintro.dto.book;
+
+public record BookSearchParametersDto(String[] title, String[] author) {
+}
```

Comment:
This is not part of the global exception handler, you should create pull request from this brach to add_criteria_query_book not to main.
Check this out https://mate-academy.github.io/jv-program-fulltime/07_spring_boot/git-strategy